packages/fuels-programs/src/debug.rs

```diff
@@ -156,6 +156,7 @@ fn parse_loader_script(script: &[u8], data: &[u8]) -> Result<Option<(ScriptCallD
         ScriptCallData {
             code: script.to_vec(),
             data: data.to_vec(),
+            #[allow(deprecated)]
```

Comment:
I would not deprecate this rather call it `section_offset` or `split_section_offset` or something else. Tooling still needs this information to shift the the old configurables.